### ISSUE TITLE
The collapse-expand button appears when 

### ISSUE THREAD

**Issue by tlle14** [linagora/esn-frontend-inbox#491]
# Precondition
The email has an attachments

# Actual Result
The collapse-expand arrow appears even when the attachment list has a few.
It is clickable but doesn't work. 
![Peek 2021-06-24 16-23](https://user-images.githubusercontent.com/69950655/123238798-fa41c100-d508-11eb-9640-97e01b2b1dbb.gif)

# Expected Result
The button should appear for the long attachments list

# Environment
DEV platform
Production

**Comment by fabienmoyon** [comment]
yes, I will try to improve it by display it only if the is a scrollbar in the attachments section
